### PR TITLE
Enable pdb in a docker container

### PR DESCRIPTION
**Pull request by sarah-clements**
I finally got a python debugging tool to work with docker (and it was painless)!!

FYI @camd and @jmaher

### CODE REVIEW

File: docs/common_tasks.md [hunk 1]
```diff
@@ -41,3 +41,10 @@ To add a new repository, the following steps are needed:
 
 For more information on adding a new GitHub repository, see
 [Adding a GitHub repository](submitting_data.md#adding-a-github-repository).
+
+## Debugging Tools
+
```
Comment (sarah-clements) [inline docs/common_tasks.md]: Wondering if this should go in a separate section. Common tasks seems to be a catch-all and might be missed by newcomers.

File: docs/common_tasks.md [hunk 2]
```diff
@@ -41,3 +41,11 @@ To add a new repository, the following steps are needed:
...
+You can use the Python Debugger ([pdb](https://docs.python.org/3.7/library/pdb.html)) in a docker container.
```
Comment (armenzg) [inline docs/common_tasks.md]: Please capitalize Docker and Treeherder (next line).

File: docs/common_tasks.md (hunk 1 above)
Comment (armenzg) [reply docs/common_tasks.md]: I agree. It's something I've been thinking for a while. We're probably overdue to overhaul the documentation.

File: docs/common_tasks.md [hunk 3]
```diff
@@ -41,3 +41,11 @@ To add a new repository, the following steps are needed:
...
+After starting a local treeherder instance using [docker-compose](installation.md#server-and-full-stack-development),
+in a separate shell type `docker attach backend`. Then set a breakpoint in your file using either `import pdb; pdb.set_trace()`
```
Comment (armenzg) [inline docs/common_tasks.md]: That's really simple. Thank you for finding out this trick!

Comment (sarah-clements) [reply docs/common_tasks.md]: I'm glad you mentioned pdb last week when I was asking you about venv and vscode debugging. :)

File: docs/common_tasks.md (hunk 2 above)
Comment (sarah-clements) [reply docs/common_tasks.md]: Will do.

**Pull request merged**